clang/test/AST/Interp/records.cpp

```diff
@@ -599,7 +599,7 @@ namespace Destructors {
   };
   constexpr int testS() {
     S{}; // ref-note {{in call to 'S{}.~S()'}} \
-         // expected-note {{in call to '&S{}->~S()'}}
+         // expected-note {{in call to 'S{}.~S()'}}
```

Comment:
```suggestion
         // both-note {{in call to 'S{}.~S()'}}
```
and remove the line above if the output matches anyway.

Reply:
Done.